### PR TITLE
#38 update Java to Jakarta in the ApplicationAssemblyDeployment section

### PR DESCRIPTION
**Pull request by Emily-Jiang**
Signed-off-by: Emily Jiang <emijiang6@googlemail.com>

### CODE REVIEW

File: specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc [hunk 1]
```diff
@@ -60,47 +59,45 @@ image:Platform_Spec-10.png[image]
 The EJB, servlet, application client, and
```
Comment (smillidge) [inline specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc]: These specs need renaming to `Jakarta Enterprtise Beans, Jakarta Servlet, application client and Jakarta Connectors`

File: specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc [hunk 2]
```diff
@@ -60,47 +59,45 @@ image:Platform_Spec-10.png[image]
...
-architecture required to produce Java EE modules. (The application
+architecture required to produce Jakarta EE modules. (The application
 client specification is found in
 link:#a3293[See CHApter] of this
 document.)
 
-A Java EE module is a collection of one or more
-Java EE components (web, EJB, application client, or Connector) with an
+A Jakarta EE module is a collection of one or more
+Jakarta EE components (web, EJB, application client, or Connector) with an
```
Comment (smillidge) [inline specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc]: references EJB

File: specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc [hunk 3]
```diff
@@ -60,47 +59,45 @@ image:Platform_Spec-10.png[image]
...
 optional module deployment descriptor of that type. Any number of
 components of the same container type can be packaged together with a
-single Java EE deployment descriptor appropriate to that container type
-to produce a Java EE module. Components of different container types may
-not be mixed in a single Java EE module, except for the packaging of EJB
+single Jakarta EE deployment descriptor appropriate to that container type
+to produce a Jakarta EE module. Components of different container types may
+not be mixed in a single Jakarta EE module, except for the packaging of EJB
```
Comment (smillidge) [inline specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc]: references EJB

File: specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc [hunk 4]
```diff
@@ -60,47 +59,45 @@ image:Platform_Spec-10.png[image]
...
+be composed of multiple Jakarta EE modules.
+* The deployment descriptor for a Jakarta EE
 module contains declarative data required to deploy the components in
-the module. The deployment descriptor for a Java EE module also contains
+the module. The deployment descriptor for a Jakarta EE module also contains
 assembly instructions that describe how the components are composed into
 an application.
-* Starting with version 5 of the Java EE
-platform, a web application module, an enterprise bean module, or an
+* Starting with version 5 of the Jakarta EE, a web application module, an enterprise bean module, or an
```
Comment (smillidge) [inline specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc]: Question here as this talks about version 5 maybe this should remain as Java EE

File: specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc [hunk 5]
```diff
@@ -60,47 +59,45 @@ image:Platform_Spec-10.png[image]
...
 application client module need not contain a deployment descriptor.
 Instead, the deployment information may be specified by annotations
 present in the class files of the module.
-* Starting with version 5 of the Java EE
-platform, a Java EE enterprise application archive need not contain a
+* Starting with version 5 of the Jakarta EE, a Jakarta EE enterprise application archive need not contain a
```
Comment (smillidge) [inline specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc]: same as comment above

File: specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc [hunk 6]
```diff
@@ -354,16 +351,16 @@ installed library, the library must be made available to all components
 in all _.war_ files, EJB _.jar_ files, application _.jar_ files, and
```
Comment (smillidge) [inline specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc]: EJB should be replaced

File: specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc [hunk 7]
```diff
@@ -594,17 +591,17 @@ link:#a2966[See Dynamic Class Loading]”
...
+below, a Jakarta EE product must provide a way to allow an application to
 access a class library installed in the application server, even if it
 has not expressed a dependency on that library. This supports the use of
 old applications and extension libraries that do not use the defined
 extension dependency mechanism.
 
 The following sections describe the
 requirements for each container type. In all cases, access to classes is
-governed by the rules of the Java language and the Java virtual machine.
+governed by the rules of the Jakarta language and the Jakarta virtual machine.
```
Comment (smillidge) [inline specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc]: This should still be Java language and Java virtual machine

File: specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc [hunk 8]
```diff
@@ -594,17 +591,17 @@ link:#a2966[See Dynamic Class Loading]”
...
 In all cases, access to classes and resources is governed by the rules
-of the Java security model.
+of the Jakarta security model.
```
Comment (smillidge) [inline specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc]: this should probably remain as Java as well

File: specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc [hunk 9]
```diff
@@ -617,7 +614,7 @@ if libraries need to access classes packaged in the application modules.
 Components in the web container must have
 access to the following classes
 
-and resources. Note that as of Java EE 6, EJB
+and resources. Note that as of Jakarta EE 6, EJB
```
Comment (smillidge) [inline specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc]: Maybe this should say Java EE 6 as it is an older version being referred to. Although EJB needs to change

File: specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc [hunk 10]
```diff
@@ -617,7 +614,7 @@ if libraries need to access classes packaged in the application modules.
...
 components may be packaged in a web component module. Such EJB
 components have the same access as other components in the web
 container. See the EJB specification for further detail.
```
Comment (smillidge) [inline specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc]: Replace EJB references

File: specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc [hunk 11]
```diff
@@ -735,8 +732,8 @@ link:#a2945[See Library Support]”.
 * The transitive closure of any libraries
 referenced by the rar files themselves (as specified in
 link:#a2945[See Library Support]”).
-* The Java EE API classes specified in
-link:#a2159[See Java EE
+* The Jakarta EE API classes specified in
+link:#a2159[See Jakarta EE
 Technologies] for the EJB container.
```
Comment (smillidge) [inline specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc]: Replace EJB

File: specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc [hunk 12]
```diff
@@ -767,8 +764,8 @@ link:#a2945[See Library Support]”).
 * The transitive closure of any libraries
 referenced by the rar files above themselves (as specified in
 link:#a2945[See Library Support]”).
-* The Java EE API classes specified in
-link:#a2159[See Java EE
+* The Jakarta EE API classes specified in
+link:#a2159[See Jakarta EE
 Technologies] for the containers other than the EJB container.
```
Comment (smillidge) [inline specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc]: replace EJB

File: specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc [hunk 13]
```diff
@@ -887,7 +884,7 @@ component dependency, there must only be one corresponding component
 that fulfills that dependency in the scope of the application.
 . For each _ejb-link_ , there must be only one
 matching _ejb-name_ in the scope of the entire application (see
-link:#a936[See Enterprise JavaBeans™
+link:#a936[See Enterprise JakartaBeans™
 (EJB) References]”).
```
Comment (smillidge) [inline specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc]: Is EJB retained here?

File: specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc [hunk 14]
```diff
@@ -901,19 +898,19 @@ unique names. Descriptions of role-names that are used by many
...
+included in the Jakarta EE application. The context root is a relative name
 in the web namespace for the application. Each web module must be given
 a distinct and non-overlapping name for its context root. The web
 modules will be assigned a complete name in the namespace of the web
-server at deployment time. If there is only one web module in the Java
+server at deployment time. If there is only one web module in the Jakarta
 EE application, the context root may be the empty string. If no
 deployment descriptor is included in the application package, it will
 use the default-context-path in the web module. Otherwise, it will use
 the module name as the context root of the web module. See the Servlet
```
Comment (smillidge) [inline specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc]: Jakarta Servlet specification?

File: specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc [hunk 15]
```diff
@@ -933,15 +930,15 @@ undesirable consequences and is not recommended.
...
+of application clients, a Jakarta EE application should not assume that
 each component is loaded in a separate class loader and has a separate
 namespace. All components in a single application may be loaded in a
 single class loader and share a single namespace. Note, however, that it
 must be possible to deploy an application such that all components of
 the application are in a namespace (or namespaces) separate from that of
 other applications. Typically, this will be the normal method of
 deployment. By default, application clients are each deployed into their
-own Java virtual machine instance, and thus each application client has
+own Jakarta virtual machine instance, and thus each application client has
```
Comment (smillidge) [inline specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc]: should remain Java

File: specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc [hunk 16]
```diff
@@ -1146,25 +1143,25 @@ Specifications that define their own additional deployment descriptors
...
 
-. The deployment tool must first read the Java
+. The deployment tool must first read the Jakarta
 EE module deployment descriptor if provided externally to the package or
 if present in the package. See the component specifications for the
 required location and name of the deployment descriptor for each
 component type.
 . If the deployment descriptor is absent, or
-is present and is a Java EE 5 or later version descriptor and the
+is present and is a Jakarta EE 5 or later version descriptor and the
```
Comment (smillidge) [inline specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc]: not sure whether this should say Java EE 5 as there is no jakarta EE 5

File: specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc [hunk 17]
```diff
@@ -1241,16 +1238,16 @@ unique name or allow the Deployer to choose a unique name, but must not
...
 be deployable.
 . The deployment tool must open each of the
-Java EE modules listed in the Java EE application deployment descriptor
-or discovered using the rules above and read the Java EE module
+Jakarta EE modules listed in the Jakarta EE application deployment descriptor
+or discovered using the rules above and read the Jakarta EE module
 deployment descriptor, if present in the package. See the Enterprise
-JavaBeans, servlet, Java EE Connector and application client
+JakartaBeans, servlet, Jakarta EE Connector and application client
 specifications for the required location and name of the deployment
```
Comment (smillidge) [inline specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc]: Should be Jakarta Connectors

File: specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc [hunk 18]
```diff
@@ -1241,16 +1238,16 @@ unique name or allow the Deployer to choose a unique name, but must not
...
 descriptor for each component type. Deployment descriptors are optional
 for all module types. (The application client specification is
 link:#a3294[See Application Clients]”.)
 . If the module deployment descriptor is
-absent, or is present and is a Java EE 5 or later version descriptor and
+absent, or is present and is a Jakarta EE 5 or later version descriptor and
```
Comment (smillidge) [inline specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc]: again not sure whether this should say Java EE 5

File: specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc [hunk 19]
```diff
@@ -1346,43 +1343,43 @@ report an error. Application client modules are initialized on their own
...
+All valid Jakarta EE application deployment
 descriptors must conform to the XML Schema definition, or the DTD or
 schema definition from a previous version of this specification. (See
 link:#a3447[See Previous Version
 Deployment Descriptors].”) The deployment descriptor must be named
 _META-INF/application.xml_ in the _.ear_ file. Note that this name is
 case-sensitive. The XML Schema located at
-_http://xmlns.jcp.org/xml/ns/javaee/application_8.xsd_ defines the XML
-grammar for a Java EE application deployment descriptor.
+_http://xmlns.jcp.org/xml/ns/jakartaee/application_8.xsd_ defines the XML
```
Comment (smillidge) [inline specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc]: not sure what URL the xsd will be at but this URL is certain to be wrong

File: specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc [hunk 20]
```diff
@@ -1346,43 +1343,43 @@ report an error. Application client modules are initialized on their own
...
 
 image:Platform_Spec-11.png[image]
 
-=== Common Java EE XML Schema Definitions
+=== Common Jakarta EE XML Schema Definitions
 
 The XML Schema located at
-_http://xmlns.jcp.org/xml/ns/javaee/javaee_8.xsd_ defines types that are
-used by many other Java EE deployment descriptor schemas, both in this
+_http://xmlns.jcp.org/xml/ns/jakartaee/jakartaee_8.xsd_ defines types that are
```
Comment (smillidge) [inline specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc]: this URL will be wrong unless the jcp is going to host our xsds!!

File: specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc (hunk 19 above)
Comment (Emily-Jiang) [reply specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc]: I realised this and reverted my changes and raised an issue https://github.com/eclipse-ee4j/jakartaee-platform/issues/45

File: specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc (hunk 20 above)
Comment (Emily-Jiang) [reply specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc]: I realised this and reverted my changes and raised an issue https://github.com/eclipse-ee4j/jakartaee-platform/issues/45

File: specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc (hunk 1 above)
Comment (Emily-Jiang) [reply specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc]: done

File: specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc (hunk 2 above)
Comment (Emily-Jiang) [reply specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc]: done

File: specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc (hunk 4 above)
Comment (Emily-Jiang) [reply specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc]: agreed. reverted.

File: specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc (hunk 5 above)
Comment (Emily-Jiang) [reply specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc]: reverted

File: specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc (hunk 6 above)
Comment (Emily-Jiang) [reply specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc]: done

File: specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc (hunk 8 above)
Comment (Emily-Jiang) [reply specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc]: agreed. reverted.

File: specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc (hunk 9 above)
Comment (Emily-Jiang) [reply specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc]: reverted.

File: specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc (hunk 11 above)
Comment (Emily-Jiang) [reply specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc]: done

File: specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc (hunk 12 above)
Comment (Emily-Jiang) [reply specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc]: done

File: specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc (hunk 13 above)
Comment (Emily-Jiang) [reply specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc]: deleted

File: specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc (hunk 14 above)
Comment (Emily-Jiang) [reply specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc]: done

File: specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc (hunk 15 above)
Comment (Emily-Jiang) [reply specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc]: reverted

File: specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc (hunk 16 above)
Comment (Emily-Jiang) [reply specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc]: agreed. reverted

File: specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc (hunk 17 above)
Comment (Emily-Jiang) [reply specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc]: done

File: specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc (hunk 18 above)
Comment (Emily-Jiang) [reply specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc]: reverted

File: specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc [hunk 21]
```diff
@@ -475,9 +472,9 @@ app1.ear:
 
  META-INF/application.xml
 
- ejb1.jar Class-Path: util.jar
+ enterprisebean1.jar Class-Path: util.jar
```
Comment (kwsutter) [inline specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc]: Since this was just an artifact of the application being described, there was no reason to change this from ejb1 and ejb2.  Just commenting, not going to hold up the approval.

File: specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc [hunk 22]
```diff
@@ -487,53 +484,53 @@ The next example illustrates a more complex
...
  META-INF/application.xml
 
- ejb1.jar Class-Path: ejb1_client.jar
+ enterprisebean1.jar Class-Path: enterprisebean1_client.jar
 
  deployment descriptor contains:
 
 
-<ejb-client-jar>ejb1_client.jar</ejb-client-jar>
+<enterprisebean-client-jar>enterprisebean1_client.jar</enterprisebean-client-jar>
```
Comment (kwsutter) [inline specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc]: This change is affecting xml elements...  I don't think we should be modifying ejb-client-jar to enterprisebean-client-jar for the deployment descriptor.  I think we have to revert these changes.

File: specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc [hunk 23]
```diff
@@ -487,53 +484,53 @@ The next example illustrates a more complex
...
- ejb3.jar Class-Path: ejb1_client.jar
+ enterprisebean3.jar Class-Path: enterprisebean1_client.jar
 
- webapp.war Class-Path: ejb1_client.jar
+ webapp.war Class-Path: enterprisebean1_client.jar
 
  WEB-INF/web.xml
 
- WEB-INF/lib/servlet1.jar
+ WEB-INF/lib/jakartaservlet1.jar
```
Comment (kwsutter) [inline specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc]: We're going too far with the changes...  The context does not require us to change all of these artifacts.

File: specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc [hunk 24]
```diff
@@ -594,15 +591,15 @@ link:#a2966[See Dynamic Class Loading]”
...
+below, a Jakarta EE product must provide a way to allow an application to
 access a class library installed in the application server, even if it
 has not expressed a dependency on that library. This supports the use of
 old applications and extension libraries that do not use the defined
 extension dependency mechanism.
 
 The following sections describe the
 requirements for each container type. In all cases, access to classes is
-governed by the rules of the Java language and the Java virtual machine.
+governed by the rules of the Jakarta language and the Java virtual machine.
```
Comment (kwsutter) [inline specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc]: It's still the Java language...  not the Jakarta language...  :-)

File: specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc [hunk 25]
```diff
@@ -617,10 +614,10 @@ if libraries need to access classes packaged in the application modules.
 Components in the web container must have
 access to the following classes
 
-and resources. Note that as of Java EE 6, EJB
-components may be packaged in a web component module. Such EJB
+and resources. Note that as of Java EE 6, Jakarta Enterprise Beans
```
Comment (kwsutter) [inline specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc]: Here the context is clearly before Java EE 8/Jakarta EE 8...  We don't need to modify every instance of EJB...

File: specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc [hunk 26]
```diff
@@ -885,10 +882,9 @@ descriptor to specify a unique name for one of the modules.
 of all components in every module contained in the application. For each
 component dependency, there must only be one corresponding component
 that fulfills that dependency in the scope of the application.
-. For each _ejb-link_ , there must be only one
-matching _ejb-name_ in the scope of the entire application (see
-link:#a936[See Enterprise JavaBeans™
-(EJB) References]”).
+. For each _enterprisebean-link_ , there must be only one
```
Comment (kwsutter) [inline specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc]: These links being referenced (here and earlier) are referring to the dependencies...  I don't think these need to change...

File: specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc [hunk 27]
```diff
@@ -1177,11 +1173,11 @@ automatically choose a unique name or allow the Deployer to choose a
...
 existing modules continue to be deployable.
 . The deployment tool must deploy all of the
-components listed in the Java EE module deployment descriptor, or marked
+components listed in the Jakarta EE module deployment descriptor, or marked
 via annotations and discovered as described in the previous requirement,
-according to the deployment requirements of the respective Java EE
+according to the deployment requirements of the respective Jakarta EE
 component specification. If the module is a type that contains JAR
-format files (for example, web and Connector modules), all classes in
+format files (for example, web and Jakarta Connector modules), all classes in
```
Comment (kwsutter) [inline specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc]: This was just a generic reference, I would have just lowercased "connector".

File: specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc [hunk 28]
```diff
@@ -1195,16 +1191,16 @@ via annotations discovered in requirement 2.
 . The deployment tool must allow the Deployer
 to deploy the same module multiple times, as multiple independent
 applications, possibly with different configurations. For example, the
-enterprise beans in an ejb-jar file might be deployed multiple times
+enterprise beans in an enterprisebean-jar file might be deployed multiple times
```
Comment (kwsutter) [inline specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc]: again, this artifact should probably be left as ejb-jar.

File: specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc [hunk 29]
```diff
@@ -1228,9 +1224,9 @@ _META-INF/MANIFEST.MF_ file with a _Main-Class_ attribute, or contains a
 _META-INF/application-client.xml_ file, consider the .jar file to be an
 application client module.
 . If the _.jar_ file contains a
-_META-INF/ejb-jar.xml_ file, or contains any class with an EJB
+_META-INF/enterprisebean-jar.xml_ file, or contains any class with an Jakarta Enterprise Beans
```
Comment (kwsutter) [inline specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc]: Don't change ejb-jar.xml

File: specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc (hunk 18 above)
Comment (rhuan080) [inline specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc]: Hi, 
In this point I think is Java and no Jakarta, because is Java EE 5. Jakarta is starting on version 8.

File: specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc (hunk 28 above)
Comment (Emily-Jiang) [reply specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc]: done

File: specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc (hunk 21 above)
Comment (Emily-Jiang) [reply specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc]: done

File: specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc (hunk 22 above)
Comment (Emily-Jiang) [reply specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc]: done

File: specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc (hunk 23 above)
Comment (Emily-Jiang) [reply specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc]: done

File: specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc (hunk 24 above)
Comment (Emily-Jiang) [reply specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc]: done

File: specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc (hunk 25 above)
Comment (Emily-Jiang) [reply specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc]: done

File: specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc (hunk 26 above)
Comment (Emily-Jiang) [reply specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc]: reverted

File: specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc (hunk 27 above)
Comment (Emily-Jiang) [reply specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc]: done

File: specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc (hunk 29 above)
Comment (Emily-Jiang) [reply specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc]: done

File: specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc (hunk 18 above)
Comment (Emily-Jiang) [reply specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc]: It was reverted in my previous commit. It was outdated.

File: specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc [hunk 30]
```diff
@@ -1177,7 +1177,7 @@ components listed in the Jakarta EE module deployment descriptor, or marked
 via annotations and discovered as described in the previous requirement,
 according to the deployment requirements of the respective Jakarta EE
 component specification. If the module is a type that contains JAR
-format files (for example, web and Jakarta Connector modules), all classes in
+format files (for example, web and Jakarta connector modules), all classes in
```
Comment (kwsutter) [inline specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc]: Drop the "Jakarta" in this case.

Comment (Emily-Jiang) [reply specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc]: done
